## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 417815`
Commit: `562fce9b6e9e0ee2b71baadad4f7e40acf9c3315`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>fcitx5-openbangla-keyboard</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>fcitx5</li>
    <li>fcitx5-anthy</li>
    <li>fcitx5-bamboo</li>
    <li>fcitx5-chewing</li>
    <li>fcitx5-gtk</li>
    <li>fcitx5-gtk.dev</li>
    <li>fcitx5-hangul</li>
    <li>fcitx5-lua</li>
    <li>fcitx5-m17n</li>
    <li>fcitx5-mcbopomofo</li>
    <li>fcitx5-mozc</li>
    <li>fcitx5-mozc-ut</li>
    <li>fcitx5-rime</li>
    <li>fcitx5-skk</li>
    <li>fcitx5-table-extra</li>
    <li>fcitx5-table-other</li>
    <li>kdePackages.fcitx5-chinese-addons (qt6Packages.fcitx5-chinese-addons)</li>
    <li>kdePackages.fcitx5-configtool (qt6Packages.fcitx5-configtool)</li>
    <li>kdePackages.fcitx5-qt (qt6Packages.fcitx5-qt)</li>
    <li>kdePackages.fcitx5-skk-qt (qt6Packages.fcitx5-skk-qt)</li>
    <li>kdePackages.fcitx5-unikey (qt6Packages.fcitx5-unikey)</li>
    <li>kdePackages.fcitx5-with-addons (qt6Packages.fcitx5-with-addons)</li>
    <li>libime</li>
    <li>mlterm</li>
    <li>mlterm-wayland</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>fcitx5-openbangla-keyboard</summary>
<pre>      |           ^~~~~~~~~~~~
/build/source/src/engine/fcitx/openbangla.cpp:366:48: error: 'StandardPath' has not been declared
  366 |           StandardPath::global().userDirectory(StandardPath::Type::Data),
      |                                                ^~~~~~~~~~~~
/build/source/src/engine/fcitx/openbangla.cpp: In member function 'virtual void fcitx::OpenBanglaEngine::reloadConfig()':
/build/source/src/engine/fcitx/openbangla.cpp:453:15: error: 'StandardPath' has not been declared
  453 |   auto path = StandardPath::global().locate(StandardPath::Type::Config,
      |               ^~~~~~~~~~~~
/build/source/src/engine/fcitx/openbangla.cpp:453:45: error: 'StandardPath' has not been declared
  453 |   auto path = StandardPath::global().locate(StandardPath::Type::Config,
      |                                             ^~~~~~~~~~~~
/build/source/src/engine/fcitx/openbangla.cpp:464:21: error: 'StandardPath' has not been declared
  464 |   auto configFile = StandardPath::global().open(
      |                     ^~~~~~~~~~~~
/build/source/src/engine/fcitx/openbangla.cpp:465:7: error: 'StandardPath' has not been declared
  465 |       StandardPath::Type::Config, "OpenBangla/Keyboard.conf", O_RDONLY);
      |       ^~~~~~~~~~~~
make[2]: *** [src/engine/fcitx/CMakeFiles/openbangla.dir/build.make:79: src/engine/fcitx/CMakeFiles/openbangla.dir/openbangla.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:342: src/engine/fcitx/CMakeFiles/openbangla.dir/all] Error 2
make: *** [Makefile:156: all] Error 2</pre>
</details>
</details>

--- a/pkgs/by-name/li/libime/package.nix
+++ b/pkgs/by-name/li/libime/package.nix
@@ -22,21 +22,21 @@ let
     url = "https://download.fcitx-im.org/data/lm_sc.arpa-${arpaVer}.tar.zst";
     hash = "sha256-7oPs8g1S6LzNukz2zVcYPVPCV3E6Xrd+46Y9UPw3lt0=";
   };
-  dictVer = "20241001";
+  dictVer = "20250327";
   dict = fetchurl {
     url = "https://download.fcitx-im.org/data/dict-${dictVer}.tar.zst";
-    hash = "sha256-0zE7iKaGIKI7yNX5VkzxtniEjcevVBxPXwIZjlo2hr8=";
+    hash = "sha256-fKa+R1TA1MJ7p3AsDc5lFlm9LKH6pcvyhI2BoAU8jBM=";
   };
 in
 stdenv.mkDerivation rec {
   pname = "libime";
-  version = "1.1.10";
+  version = "1.1.11";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = "libime";
     tag = version;
-    hash = "sha256-liVJEBUYcVYjjJCMW68xXbEHKQpAgTLCPm2yIdWG3IQ=";
+    hash = "sha256-C9l7VBSUdSpnt+8ghdmLljZXHFswTyi/ItqeeYTjF4Y=";
     fetchSubmodules = true;
   };
 

